events.ts

```diff
@@ -36,9 +50,19 @@ const cover = {
     if (wrapper) {
       if (options.init) {
```

Comment:
I guess I can make options default as an object instead of using an if,
`waitFor: (selector, callback, options = {}) => {`

Another solution is to make all calls to `options.init` etc like `options?.init` right?